content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -164,11 +172,16 @@ Migrations can be only be run on a direct (non-proxied) database connection. Ass
 }
 ```
 
-### Deploying to Vercel
+### Deploying to Vercel Serverless Functions
+
+You only need minimal changes to an existing project using Prisma Client with [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions#serverless-functions) to start using the Prisma Data Proxy. 
```

Comment:
```suggestion
To start using the Prisma Data Proxy, you need to make only minimal changes to an existing project, using Prisma Client with [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions#serverless-functions).
```

Reply:
I already accepte a change from Andrew on that. Can you please check again if the current version is ok - and if not make that suggestion on the current one again? Thanks.

Reply:
I trust Andrew. :-)

Reply:
Current state:
You only need minimal changes to an existing project using Prisma Client with [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions#serverless-functions) to start using the Prisma Data Proxy.